### DISCUSSION TITLE
index.vue:62 Couldn't initiate Virtual Select

### DISCUSSION THREAD

**Discussion by toyota-167** [sa-si-dev/virtual-select#332] [Q&A]
why i get the warning in the browser console: index.vue:62 Couldn't initiate Virtual Select
![image](https://github.com/sa-si-dev/virtual-select/assets/32330164/221e82be-d466-445d-a8e6-e091f96a3d54)
![image](https://github.com/sa-si-dev/virtual-select/assets/32330164/abb835b1-6d61-4ffd-984f-d1b0ba36dbd6)
![image](https://github.com/sa-si-dev/virtual-select/assets/32330164/3f1a680b-cc8f-4bda-aa81-aaa25b531433)

**Comment by kjmitch** [comment]
Try supplying the `ele` property with an element instead of selector text. I don't know how it's actually written but the initialization method may only know how to use the HTML Element object returned by the `.querySelector()` method, and not have a way to use the string you're giving it (even if it is a valid selector string).

So instead of

``` javascript
VirtualSelect.init(
{
    ele: '#sample-select',
    ...
} );
```

try

``` javascript
let sampleSelect = document.querySelector( '#sample-select' );
VirtualSelect.init(
{
    ele: sampleSelect,
    ...
} );`
```

and see if that solves your issue.

**Comment by gnbm** [comment]
@avida-1906, please share a CodePen (or similar) with your use case. And share what you've explored so far, so the community can try to help
